### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tlegen-k**
Updated README to reflect recent changes

## Related Issues and Dependencies

…

## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

… Explain your changes.

## Description

Please let me know if this is good enough or there is something else that I can improve.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@
 This is a Template for any Python based project, it contains what Project Thoth and the AI CoE need:
 
 1. GitHub defaults and Templates for issues
-2. configuration for Coala and Black (code formating)
-3. basic configuration for Zuul
-4. configuration for Thoth (stage environment, Red Hat VPN only)
+2. configuration for Coala and Black (code formating) are done using `prow` with pre-commit and `mypy`.
```
Comment (pacospace) [inline README.md:6]: `pre-commit` and maybe we can add links to prow?

File: README.md [hunk 2]
```diff
@@ -3,9 +3,9 @@
...
+3. Zuul is no longer suppported. Instead AICOE-CI is used for basic configuration.
```
Comment (pacospace) [inline README.md:7]: Maybe we can add link to AICOE-CI: https://github.com/AICoE/aicoe-ci

File: README.md (hunk 1 above)
Comment (pacospace) [reply README.md:6]: @goern @harshad16  what are best docs for prow to link here?

File: README.md [hunk 3]
```diff
@@ -3,9 +3,9 @@
...
+4. we have a public service configuration now accessible for everyone.
```
Comment (pacospace) [inline README.md:8]: We

**Review by pacospace — COMMENTED** [review]
Very nice!! Few comments

File: README.md (hunk 1 above)
Comment (tlegen-k) [reply README.md]: added link to prow repository

File: README.md [hunk 4]
```diff
@@ -3,10 +3,10 @@
...
 1. GitHub defaults and Templates for issues
-2. configuration for Coala and Black (code formating)
-3. basic configuration for Zuul
-4. configuration for Thoth (stage environment, Red Hat VPN only)
-5. if you are writing a Python module, Kebechet could manage releases of your packages for you
-6. if credentials are provided, Python module releases could be published to PyPI
+2. Configuration for Coala and Black (code formating) are done using [`prow`](https://github.com/kubernetes/test-infra/tree/master/prow) with `pre-commit` and `mypy`.
+3. Zuul is no longer suppported. Instead [AICOE-CI](https://github.com/AICoE/aicoe-ci) is used for basic configuration.
+4. We have a public service configuration now accessible for everyone.
+5. If you are writing a Python module, Kebechet could manage releases of your packages for you
```
Comment (pacospace) [inline README.md:9]: let's link the bot for 5. and 6. https://github.com/marketplace/khebhut

Comment (tlegen-k) [reply README.md]: Done.
Can anything else be improved?

**Review by pacospace — COMMENTED** [review]
/lgtm

File: README.md (hunk 4 above)
Comment (pacospace) [reply README.md]: that was all on my mind, thanks @tlegen-k! @fridex @goern @harshad16 something more to add?

**Review by harshad16 — APPROVED** [review]

**Pull request merged**